cmp/grid/impl/SubField.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+/**
+ * @private
+ */
+export class SubField {
+    colId;
+    label;
+
+    /**
+     * @param {Object} s - SubField configuration.
+     * @param {string} s.colId - Column ID to render.
+     * @param {boolean} [c.label] - true to include label.
```

Comment:
Could we allow this to be a string, or a boolean?  String would allow you to actually specify a label?  Maybe you want/need something shorter than the column header text, or just different based on the context?